TextileNode.ts

```diff
@@ -50,6 +50,7 @@ export default {
 
   getProfile: async function (): Promise<TT.Profile> {
     const jsonString = await TextileNode.getProfile()
+    if (jsonString === '') Error('getProfile returned an empty result')
```

Comment:
@asutula before going down this path too far. is this what you were imagining?

Reply:
Not quite, i was imagining the function would return `Promise<TT.Profile | undefined>` and you would just return `undefined` in the empty string case.

Reply:
k, sounds good

Reply:
This could also be handled at the native code level where you would fulfill the promise with `nil` or `null`, but maybe it's better here since you just have to make the change in one place?

Reply:
doesn't matter really, i felt a bit like reflecting the exact api result through the bridge was more useful going forward, putting editorial in the js. either way. easy to throw in here and move down the stack if useful.